visualization/dashboard/src/MLIDashboard/JointDataset.ts

```diff
@@ -121,6 +121,24 @@ export class JointDataset {
         return result;
     }
 
+    // set the appropriate error value in the keyed column
+    public static setErrorMetrics(row: {[key: string]: any}, modelType: ModelTypes): void {
+        if (modelType === ModelTypes.regression) {
+            row[JointDataset.RegressionError] = row[JointDataset.TrueYLabel] - row[JointDataset.PredictedYLabel];
+            return;
+        }
+        if (modelType === ModelTypes.binary) {
```

Comment:
nit: change to else, remove returns

Reply:
Hmm, I actually prefer returns and ifs, is there a reason to prefer else? I think its clearer and less error prone to have explicit ifs with returns?

Reply:
It's easier to follow if/else if/else.  There are also fewer lines of code.  Following code from top to bottom without earlier returns in places is easier to follow.  Up to you to change it though.